models/action.go

```diff
@@ -629,6 +630,60 @@ func CommitRepoAction(opts CommitRepoActionOptions) error {
 			}
 		}
 
+		// if there is a pull request fro this branch, then update pull request
+		prs, err := GetUnmergedPullRequestsByHeadInfo(repo.ID, refName)
+		if err != nil {
+			return fmt.Errorf("GetUnmergedPullRequestsByHeadInfo: %v", err)
+		}
+
+		if len(prs) > 0 {
+			var isForcePush = false
+
+			// detect force push
+			if !isNewBranch && git.EmptySHA != opts.OldCommitID {
+				output, err := git.NewCommand("rev-list", opts.OldCommitID, "^"+opts.NewCommitID).RunInDir(repo.RepoPath())
+				if err != nil {
+					return fmt.Errorf("rev-list: %v", err)
+				} else if len(output) > 0 {
+					isForcePush = true
```

Comment:
`return` already interrupts the flow - so using the else if is unneccessary (remove else). Also, remove the if entirely and make it `isForcePush = len(output) > 0`?